src/graph/rome_models.cc

```diff
@@ -1133,6 +1136,9 @@ ncclResult_t parseGraph(const char* str, struct ncclTopoSystem* system, struct n
   int net_offset = 0, net_count = 0;
   int ngpus = system->nodes[GPU].count;
   int nnets = system->nodes[NET].count;
+
+  if (rcclParamModelReversalDisable) reverse = 0;
```

Comment:
Isn't this supposed to be a function call like `rcclParamModelReversalDisable`?

Reply:
> Isn't this supposed to be a function call like `rcclParamModelReversalDisable()`?

agree

Reply:
Good catch - I didn't notice it because it compiles because test for function always returns true.